exporters/etw/README.md

```diff
@@ -0,0 +1,201 @@
+# Getting Started with OpenTelemetry C++ SDK and ETW exporter on Windows
+
+Event Tracing for Windows (ETW) is an efficient kernel-level tracing facility that lets you log kernel or
+application-defined events to a log file. You can consume the events in real time or from a log file and
+use them to debug an application or to determine where performance issues are occurring in the application.
+
+OpenTelemetry C++ SDK ETW exporter allows the code instrumented using OpenTelemetry API to forward events
+to out-of-process ETW listener, for subsequent data recording or forwarding to alternate pipelines and
+flows. Windows Event Tracing infrastructure is available to any vendor or application being deployed on
+Windows.
+
+## API support
+
+These are the features planned to be supported by ETW exporter:
+
+- [x] OpenTelemetry Tracing API and SDK headers are **stable** and moving towards GA.
+- [ ] OpenTelemetry Logging API is work-in-progress, pending implementation of [Latest Logging API spec here](https://github.com/open-telemetry/oteps/pull/150)
+- [ ] OpenTelemetry Metrics API is not implemented yet.
+
+Implementation of OpenTelemetry C++ SDK ETW exporter on Windows OS is `header only` :
+
+- full definitions of all macros, functions and classes comprising the library are visible to the compiler
+in a header file form.
+- implementation does not need to be separately compiled, packaged and installed in order to be used.
+
+All that is required is to point the compiler at the location of the headers, and then `#include` the header
+files into the application source. Compiler's optimizer can do a much better job when all the library's
+source code is available. Several options below may be turned on to optimize the code with the usage of
+standard C++ library, Microsoft Guidelines Support library, Google Abseil Variant library. Or enabling
+support for non-standard features, such as 8-bit byte arrays support that enables performance-efficient
+representation of binary blobs on ETW wire.
+
+## Example project
+
+The following include directories are required, relative to the top-level source tree of OpenTelemetry C++ SDK:
+
+- api/include/
+- exporters/etw/include/
+- sdk/include/
+
+Code that instantiates ETW TracerProvider, subsequently obtaining a Tracer bound to `OpenTelemetry-ETW-Provider`,
+and emitting a span named `MySpan` with attributes on it, as well as `MyEvent` within that span.
+
+```cpp
+
+#include <map>
```

Comment:
Could the sample code be separated into .cc source file with build scripts support, and insert the link here? In that way we make sure the sample code builds fine and is up-to-date.

Reply:
This is intentional. We already have about this same code in a test app..
So I thought it'd be nice to have it embedded into markdown document.